### PR TITLE
log/jetty.server: only error

### PR DESCRIPTION
**Pull request by avelino**
related:
- #133 
- https://github.com/moclojer/moclojer/pull/135#issuecomment-1750681077

### CODE REVIEW

File: dev/moclojer/build.clj [hunk 1]
```diff
@@ -37,6 +37,7 @@
           (string/join "\n" ["-H:Name=moclojer"
                              "-Dio.pedestal.log.defaultMetricsRecorder=nil"
                              "-Dorg.slf4j.simpleLogger.defaultLogLevel=warn"
+                             "-Dorg.slf4j.simpleLogger.log.org.eclipse.jetty.server=error"
```
Comment (matheusfrancisco) [inline dev/moclojer/build.clj:40]: if I got the @souenzzo idea, with this it means we don't need the custom log to remove info logs from jetty, right? but this is only once we build, it needs to added this in a way when we are developing use it also.

Comment (matheusfrancisco) [reply dev/moclojer/build.clj:40]: and with this dependency we can remove this? https://github.com/moclojer/moclojer/pull/135/files#diff-69e4b05396b1e3c99379e0e2b572d088f1ef355a6530d92f735a7157b9134d44R9 or i got the wrong idea?

Comment (souenzzo) [reply dev/moclojer/build.clj:40]: to use this solution, it is required:
- remove the code that you highlighetd
- remove [org.slf4j/slf4j-jdk14](https://github.com/moclojer/moclojer/blob/main/deps.edn#L12C25-L12C25)
- optional: bump [org.slf4j/slf4j-simple](https://mvnrepository.com/artifact/org.slf4j/slf4j-simple) to `v2.x` (I didn't study the API v1. Maybe it do not include those extra settings) 


btw, note that it is not a required move. the code currently work as is. and it is not wrong.

I was just sharing an alternative way to do :)

Comment (avelino) [reply dev/moclojer/build.clj:40]: @souenzzo which way is simpler to implement (easier to understand)?
if it's the new way you mentioned, can you open a PR with the implementation?

**Pull request closed without merging**